src/constants/hypixel/Hypixel.js

```diff
@@ -260,4 +260,221 @@ export const HYPIXEL = {
 		'stats.SkyClash.wins', // SkyClash (removed)
 		'stats.WoolGames.wool_wars.stats.wins',
 	],
+	MODES: {
+		ARCADE: {
+			"DAYONE": 'Blocking Dead',
+			"ONEINTHEQUIVER": 'Bounty Hunters',
+			"PVP_CTW": 'Capture the Wool',
+			"DEFENDER": 'Creeper Attack',
+			"DRAGONWARS2": 'Dragon Wars',
+			"DROPPER": 'Dropper',
+			"ENDER": 'Ender Spleef',
+			"FARM_HUNT": 'Farm Hunt',
+			"SOCCER": 'Football',
+			"STARWARS": 'Galaxy Wars',
+			"HIDE_AND_SEEK_PROP_HUNT": 'Hide and Seek: Prop Hunt',
+			"HIDE_AND_SEEK_PARTY_POOPER": 'Hide and Seek: Party Pooper',
+			"HOLE_IN_THE_WALL": 'Hole in the Wall',
+			"SIMON_SAYS": 'Hypixel Says',
+			"MINI_WALLS": 'Mini Walls',
+			"PARTY": 'Party Games',
+			"DRAW_THEIR_THING": 'Pixel Painters',
+			"PIXEL_PARTY": 'Pixel Party',
+			"THROW_OUT": 'Throw Out',
+			"ZOMBIES_DEAD_END": 'Zombies',
+			"ZOMBIES_BAD_BLOOD": 'Zombies',
+			"ZOMBIES_ALIEN_ARCADIUM": 'Zombies',
+			"ZOMBIES_PRISON": 'Zombies',
+		},
+		ARENA: {
+			"1v1": '1v1',
+			"2v2": '2v2',
+			"4v4": '4v4',
+		},
+		BATTLEGROUND: {
+			"mini_ctf": 'Capture the Flag',
+			"domination": 'Domination',
+			"team_deathmatch": 'Team Deathmatch',
+		},
+		BEDWARS: {
+			'BEDWARS_EIGHT_ONE': 'Solo',
+			'BEDWARS_EIGHT_TWO': 'Doubles',
+			"BEDWARS_EIGHT_TWO_TOURNEY": "Doubles (Tournament)",
```

Comment:
```suggestion
			"BEDWARS_EIGHT_TWO_TOURNEY": 'Doubles (Tournament)',
```
Please only use single-quotes (`'`) for values in this file.

Reply:
Got it.